packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -11,7 +11,6 @@ import { DeleteResponseCodes } from "./functions/delete-response-codes"
 import { getCollectionOnlyHasValueAndNextLink } from "./functions/get-collection-only-has-value-and-next-link"
 import { getResponseCodes } from "./functions/get-response-codes"
 import hasApiVersionParameter from "./functions/has-api-version-parameter"
-import hasheader from "./functions/has-header"
```

Comment:
Also remove has-header.cs if it's no longer used

https://github.com/Azure/azure-openapi-validator/blob/main/packages/rulesets/src/spectral/functions/has-header.ts

Reply:
never mind, i see it is still used